pydis_site/apps/content/resources/guides/pydis-guides/contributing.md

```diff
@@ -91,26 +79,55 @@ Our projects on Python Discord are open source and [available on Github](https:/
+
+  * [**Sir Lancebot**](./sir-lancebot/)
+
+  * [**Python Bot**](./bot/)
+
+  * [**Site**](./site/)
+
+### 3. Read our Contributing Guidelines
+  We have a few short rules that all contributors must follow. Make sure you read and follow them while working on our projects.
+
+  [**Contributing Guidelines**](./contributing-guidelines/).
+
+  As mentioned in the Contributing Guidelines, we have a simple style guide for our projects based on PEP 8. Give it a read to keep your code consistent with the rest of the codebase.
 
-### Useful Resources
+  [**Style Guide**](./style-guide/)
 
-[Guidelines](./contributing-guidelines/) - General guidelines you should follow when contributing to our projects.<br>
-[Style Guide](./style-guide/) - Information regarding the code styles you should follow when working on our projects.<br>
-[Review Guide](../code-reviews-primer/) - A guide to get you started on doing code reviews.
+### 4. Create an issue
+  The first step to any new contribution is an issue describing a problem with the current codebase or proposing a new feature. All the open issues are viewable on the GitHub repositories, for instance here is the [issues page for Sir Lancebot](https://github.com/python-discord/sir-lancebot/issues). If you have an idea that you want to implement, open a new issue (and check out our [**guide on writing an issue**](./issues/)). Otherwise you can browse the unassigned issues and ask to be assigned to one that you're interested in, either in the comments on the issue or in the[`#dev-contrib`](https://discord.gg/2h3qBv8Xaa)channel on Discord.
 
-## Contributors Community
-We are very happy to have many members in our community that contribute to [our open source projects](https://github.com/python-discord/).
-Whether it's writing code, reviewing pull requests, or contributing graphics for our events, it’s great to see so many people being motivated to help out.
-As a token of our appreciation, those who have made significant contributions to our projects will receive a special **@Contributors** role on our server that makes them stand out from other members.
-That way, they can also serve as guides to others who are looking to start contributing to our open source projects or open source in general.
+  Don't move forward until your issue is approved by a Core Developer. Issues are not guaranteed to be approved so your work may be wasted.
+  {: .notification .is-warning }
 
-#### Guidelines for the @Contributors Role
+### 5. Open a pull request
+  After your issue has been approved and you've written your code and tested it, it's time to open a pull request. Pull requests are a feature in GitHub; you can think of them as asking the project maintainers to accept your changes. This gives other contributors a chance to review your code and make any needed changes before it's merged into the main branch of the project.
```

Comment:
Can we mention somewhere here (or elsewhere where it makes more sense) that the commits should be made in a new branch on the fork, to simplify their contributions so that their `main` can be kept up-to-date with outs?

Reply:
I'm trying to think about the right place to put this. I don't want them to get to the PR step and then say "btw you should've been committing to a feature branch". 

I'm thinking about maybe adding a step before the PR one that includes creating a feature branch, making local commits, and pushing them to their fork.

Reply:
I think that would be good, there would be a good place to remind the user about the commit message guidelines as well!

Reply:
Added a "Making changes" step that tells them to create a feature branch and links to the commit messages guide